pkg/pushsync/pushsync.go

```diff
@@ -25,40 +25,35 @@ const (
 	streamName      = "pushsync"
 )
 
+type PushSyncer interface {
+	SendChunkAndReceiveReceipt(ctx context.Context, peer swarm.Address, ch swarm.Chunk) (*pb.Receipt, error)
```

Comment:
I suggest to rename the method to `PushChunk` and interface to `Pusher` or `ChunkPusher`. Receipts should be implicit

Reply:
The interface name cannot be Pusher as it i will confuse with the pusher pkg we will already have.  May be i can rename the function to ChunkPusher.